app/views/analytics/facilities/show.csv.erb

```diff
@@ -14,21 +14,28 @@
   'Uncontrolled rate',
   'Default rate'
 ] %>
-<% @cohort_analytics.each_with_index do |(report_dates, analytics), index| %>
+<% @cohort_analytics.each_with_index do |(report_dates, analytics), _| %>
   <%
     cohort_start, report_start = report_dates
-    registered = analytics.dig(:registered, :total) || 0
+    registered = analytics.dig(:cohort_patients, :total) || 0
```

Comment:
This change is taken care of in another PR right?

Reply:
Same, yeah.